libs/ledger-live-common/src/nft/helpers.ts

```diff
@@ -15,8 +15,10 @@ import type {
 import { API, apiForCurrency } from "../api/Ethereum";
 
 export const nftsFromOperations = (ops: Operation[]): ProtoNFT[] => {
+  const opIds = new Set();
   const nftsMap = ops
-    .slice(0)
+    // Remove any duplicate operation that could come from concatenating operations and pending operations
+    .filter((op) => (opIds.has(op.id) ? false : opIds.add(op.id)))
```

Comment:
ok i can see you did the filtering at this stage, why not but still relevant feedback on my comments above. maybe it works this way & due to fact we have to have this .nfts in the model for some reason

Reply:
Yeah I don't really how else I can find the temporary list of nfts from ops and pending ops since we're generating it at the sync level 😕 
Or else we'll need a new logic for the usage of `account.nfts` which will need to be reconcialliated as well with the pending ops to determine the real amount of an ERC1155. 

And because of OpenSea's fake transfer events, we would need 2 seperated logics to infer nfts from operations:
- 1 logic with a fail safe for negative operation when the initial amount is 0 (like [here](https://github.com/LedgerHQ/ledger-live/blob/4e17fd7aef63f40e1f142dbeec479dc219bded6c/libs/ledger-live-common/src/nft/helpers.ts#L65)) 
- 1 logic capable of returning negative balances

And then merge the arrays of nfts ?